MapboxNavigation/NavigationMapView.swift

```diff
@@ -629,13 +624,10 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
                 source.shape = arrowShape
             } else {
                 arrow.minimumZoomLevel = minimumZoomLevel
-                arrow.lineCap = MGLStyleValue(rawValue: cap)
-                arrow.lineJoin = MGLStyleValue(rawValue: join)
-                arrow.lineWidth = MGLStyleValue(interpolationMode: .exponential,
-                                                cameraStops: MBRouteLineWidthByZoomLevel.multiplied(by: 0.70),
-                                                options: [.defaultValue : MGLConstantStyleValue<NSNumber>(rawValue: 1.5)])
-                arrow.lineColor = MGLStyleValue(rawValue: .white)
-                
+                arrow.lineCap = NSExpression(forConstantValue: "cap")
+                arrow.lineJoin = NSExpression(forConstantValue: "join")
+                arrow.lineWidth = NSExpression(format: "FUNCTION($zoomLevel, 'mgl_interpolateWithCurveType:parameters:stops:', 'linear', nil, %@)", argumentArray: [MBRouteLineWidthByZoomLevel.multiplied(by: 0.70)])
```

Comment:
The syntax has now changed to `mgl_interpolate:withCurveType:parameters:stops:($zoomLevel, 'linear', nil, %@)`. (×7)